Sources_v3/StreamChat/APIClient/Endpoints/Endpoint.swift

```diff
@@ -9,7 +9,24 @@ struct Endpoint<ResponseType: Decodable> {
     let method: EndpointMethod
     let queryItems: Encodable?
     let requiresConnectionId: Bool
+    let requiresToken: Bool
     let body: Encodable?
+
+    init(
+        path: String,
+        method: EndpointMethod,
+        queryItems: Encodable? = nil,
+        requiresConnectionId: Bool = false,
+        requiresToken: Bool = true,
+        body: Encodable? = nil
+    ) {
+        self.path = path
+        self.method = method
+        self.queryItems = queryItems
+        self.requiresConnectionId = requiresConnectionId
+        self.requiresToken = requiresToken
+        self.body = body
+    }
```

Comment:
The init with default params is just a temp trick to reduce # of changes in the PR. Not sure it's the best way to go

Reply:
I think it's actually a good approach with the default param. Only a very few endpoints don't require a token.